spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+# RSpec.describe Friendship, type: :model do 
+#   context 'associations' do 
+#     it { should belongs_to(:user) } 
+#     it { should belongs_to(:friend).class_name('User') } 
+#   end 
+# end 
+
```

Comment:
## Remove commented out code
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))